File: OpenCredentialPublisher.Data/ViewModels/nG/ApiOkResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpenCredentialPublisher.Data.ViewModels.nG
{
    public class ApiOkResponse : ApiResponse
    {
        public object Result { get; }

        public ApiOkResponse(object result, string message = null, string redirectUrl = null)
            : base(200, message, redirectUrl)
        {
            Result = result;
             
        }
    }
}

File: OpenCredentialPublisher.Wallet/src/ClientApp/src/app/shared/models/apiOkResponse.ts
import { ApiResponse } from "./apiResponse";

export class ApiOkResult<T> extends ApiResponse {
  result: T;
  constructor (result: T ) {
    super(200, 'Ok', null);
    this.result = result;
  }
}

export class ApiOkResponse extends ApiResponse {
  result: any;
  constructor (result ) {
    super(200, 'Ok', null);
    this.result = result;
  }
}
